index.md

```diff
@@ -13,21 +16,34 @@ electronic structure codes, it has become possible to perform large sets
 of calculations automatically. This is the burgeoning area of
 high-throughput ab initio computation. Such calculations have been used
 to create large databases containing the calculated properties of
-existing and hypothetical materials, many of which have appeared online:
-
-- [the AFLOW distributed materials property repository](http://aflowlib.org/)
-- [the Harvard Clean Energy Project Database](http://molecularspace.org/)
-- [the Materials Cloud](http://materialscloud.org/)
-- [the Materials Project](http://materialsproject.org/)
-- [the NoMaD (Novel Materials Discovery) Repository](http://nomad-repository.eu/)
-- [the Open Quantum Materials Database](http://oqmd.org/)
-- [the Computational Materials Repository](http://cmr.fysik.dtu.dk/)
-- [the Data Catalyst Genome](http://suncat.stanford.edu/)
-- [the Materials Platform for Data Science](http://mpds.io/)
-- [the Joint Automated Repository for Various Integrated Simulations](https://jarvis.nist.gov)
-- ...
+existing and hypothetical materials, many of which have appeared online.
+
+We have released version 1.0 of the OPTIMADE specification, with several
+databases already providing implementations. A full list is available on
+the [OPTIMADE providers dashboard](https://www.optimade.org/providers-dashboard/).
+
+## How to cite OPTIMADE
 
 Should you wish to cite the OPTIMADE specification, please use the following:
 
 - Andersen *et al*, OPTIMADE: an API for exchanging materials data (2021) [arXiv:2103.02068](https://arxiv.org/abs/2103.02068){:target=_blank}
-- Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050){:target=blank}
+- Andersen *et al*, The OPTIMADE Specification, [10.5281/zenodo.4195050](https://doi.org/10.5281/zenodo.4195050){:target=_blank}
+
+## Get involved
+
+All of our work is openly available under the
+[Materials-Consortia](https://github.com/Materials-Consortia/) organization on
+GitHub and welcome all contributions.
+
+We would love to help you create and register your own OPTIMADE API
+implementations and hear your feedback on the specification.
+We meet monthly in the "OPTIMADE" room on Jitsi; everyone is welcome to
+join the discussion, please feel free to reach out on the mailing list
+`dev[at]optimade.org` to register your interest.
+The next meeting will take place on
+[Friday April 23rd at 15:00 UTC](https://www.worldtimebuddy.com/?pl=1&lid=2643743,5128581,5391959,2950159&h=2643743&date=4/23/2021%7C3&hf=1){:target=_blank}.
```

Comment:
Maybe we can just create a Google calendar and plonk all the meetings in there? That'll take care of the "rolling updates" as well...

Reply:
I would (personally) rather maintain the date on the website than a calendar, but of course we can link to it if someone else wants to set one up.